api-client/src/main/java/com/takipi/api/client/result/event/EventResult.java

```diff
@@ -89,7 +89,7 @@ public Object clone() throws CloneNotSupportedException {
 		result.introduced_by = this.introduced_by;
 		result.labels = this.labels;
 		result.similar_event_ids = this.similar_event_ids;
-		result.stats = this.stats;
+		result.stats = (MainEventStats) this.stats.clone();
```

Comment:
The cast is unnecessary, as you can defined `clone` to return the correct type

Reply:
That is super cool!! 😄